src/utils/firebase/credentials.ts

```diff
@@ -0,0 +1,22 @@
+export const APP_CREDENTIALS = {
+  apiKey: 'AIzaSyAWgOxkLC8GiPgK0Xgs_3B3LTR6MSURvrM',
+  authDomain: 'should-i-train-today.firebaseapp.com',
+  projectId: 'should-i-train-today',
+  storageBucket: 'should-i-train-today.appspot.com',
+  messagingSenderId: '756906872740',
+  appId: '1:756906872740:web:81c77f3a37e23d97fdbb31',
+  measurementId: 'G-QV0WMKXWSZ',
+  databaseUrl: 'https://should-i-train-today.firebaseio.com',
+}
```

Comment:
these are public keys so it doesnt matter to much to expose them. You set security rules based on the users `auth` object to actually protect the firebase data